xml/System.Buffers.Text/Utf8Formatter.xml

```diff
@@ -51,13 +51,27 @@
         <Parameter Name="format" Type="System.Buffers.StandardFormat" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <param name="format">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The value to format.</param>
+        <param name="destination">The buffer to write the UTF8-formatted value to.</param>
+        <param name="bytesWritten">When the method returns, contains the length of the formatted text in bytes.</param>
+        <param name="format">The standard format to use.</param>
+        <summary>Formats a <see cref="T:System.Boolean" /> as a UTF8 string.</summary>
+        <returns><see langword="true" /> if the formatting operation succeeds; <see langword="false" /> if <paramref name="buffer" /> is too small.</returns>
```

Comment:
Looks like we are missing the exceptions for all of these APIs:
https://github.com/dotnet/corefx/blob/3e36efc9a3c3f22b87c5f33a25c0ad5dbfeb9396/src/Common/src/CoreLib/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Integer.cs#L30-L32

`FormatException` for invalid `StandardFormat`.